jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -193,107 +232,12 @@ bundle exec rspec $(cat /tmp/tests-to-run)
-
-```shell
-cp -f .circleci/resources/pytest_build_config.ini pytest.ini
-```
-
-The `.circleci/resources/pytest_build_config.ini` path may need to be replaced to point to where it's located in your project.
-
-### Are you setting the junit_family in your pytest.ini?
-{: #are-you-setting-the-junit-family-in-your-pytest-ini }
-
-Check to see if you have something like `junit_family=legacy` set in your pytest.ini file. For more information on how to set `junit_family`, refer to the following page, which can be found [here](https://docs.pytest.org/en/stable/_modules/_pytest/junitxml.html). Search for "families" to see the relevant information.
-
-**Note**: A breaking change was introduced in pytest 6.1 the `junit_family` xml format changed `to xunit2`, which does not include filenames. This means that `--split-by=timings` will not work unless you specify `xunit1`. For more information see the [pytest changelog](https://docs.pytest.org/en/stable/changelog.html#id137).
-
-### Example project that correctly splits by timings
-{: #example-project-that-correctly-splits-by-timing }
-
-The example below is a fork of our `sample-python-cfd` project(https://github.com/CircleCI-Public/sample-python-cfd) that shows how you can implement test splitting:
-
-```yml
-version: 2.1
-orbs:
-  python: circleci/python@1.2
-jobs:
-  build-and-test:
-    parallelism: 2
-    docker:
-      - image: cimg/python:3.8
-    steps:
-      - checkout
-      - python/install-packages:
-          pkg-manager: pip
-      - run:
-          name: Run tests
-          command: |
-            set -e
-            TEST_FILES=$(circleci tests glob "openapi_server/**/test_*.py" | circleci tests split --split-by=timings)
-            mkdir -p test-results
-            pytest --verbose --junitxml=test-results/junit.xml $TEST_FILES
-      - store_test_results:
-          path: test-results
-      - store_artifacts:
-          path: test-results
-workflows:
-  sample:
-    jobs:
-      - build-and-test
-```
-
-You may find an example build of proper test splitting [here](https://app.circleci.com/pipelines/github/nbialostosky/sample-python-cfd/18/workflows/8b37bd45-ed19-42e1-8cc4-44401697f3fc/jobs/20)
-
-### Video: troubleshooting globbing
-{: #video-troubleshooting-globbing }
-
-Note: To follow along with the commands in the video below you will need to be [`SSH-ed into a job`]({{ site.baseurl }}/2.0/ssh-access-jobs/).
+## Using environment variables to split tests
+{: #using-environment-variables-to-split-tests }
 
-<iframe width="854" height="480" src="https://www.youtube.com/embed/fq-on5AUinE" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+For full control over how tests are split across parallel executors, CircleCI provides two environment variables that you can use in lieu of the CLI to configure each container individually.
```

Comment:
Is "in lieu" a phrase we want to use? I don't know how friendly this is for international users.